exp/api/v1beta2/awsmachinepool_types.go

```diff
@@ -171,7 +171,20 @@ type RefreshPreferences struct {
 	// The amount of capacity as a percentage in ASG that must remain healthy
 	// during an instance refresh. The default is 90.
 	// +optional
+	// +kubebuilder:validation:Minimum=0
```

Comment:
Adding new validations  to an existing field may break some users.

Just to confirm, if a user is specifying 101 as the value for this prior to this change would AWS be returning an error? (i assume so but want to check).

Reply:
I haven't tested it, but I'd assume so, [according to AWS docs](https://docs.aws.amazon.com/cli/latest/reference/autoscaling/start-instance-refresh.html). I could leave this out of this PR if you prefer.

Reply:
It may be safer...unless we explicitly test it.

Reply:
I just removed it, can you take a look again?